src/pudl/transform/ferc1.py

```diff
@@ -1851,15 +1851,15 @@ def apply_xbrl_calculation_fixes(
                         "calc_component_to_replace": {
                             "name": "nuclear_fuel_materials_and_assemblies",
                             "weight": 1.0,
-                            "source_tables": ["balance_sheet_assets_ferc1"],
+                            "source_tables": [],
```

Comment:
What does it mean to have gone from having 1 source table to zero source tables? I thought they all had to have at least one.

Reply:
to me it seems like the xbrl metadata includes a field that does not actually exist. The way i was doing the source_table before (unintentionally) defaulted to just assigning the source_table as the native table.